docs/reference/glossary.asciidoc

```diff
@@ -182,8 +179,11 @@
 
 [[glossary-type]] type ::
 
-  A type is like a _table_ in a relational database. Each type has a
+  A type represents the _type_ of document, e.g. an `email`, a `user`, or a `tweet`.
+  An <<glossary-index,index>> can contain multiple types, and each type has a
   list of <<glossary-field,fields>> that can be specified for
-  <<glossary-document,documents>> of that type. The <<glossary-mapping,mapping>>
-  defines how each field in the document is analyzed.
+  <<glossary-document,documents>> of that type.  The fields in a type are not
```

Comment:
I don't like "The fields in a type are not independent of each other" because I want to say "the fields in an __index__ are not independent of each other" but that doesn't sound right either. Maybe flip it some "fields with the same name in different types are the same field - they must have the same mapping and share the same metadata and data structures." I dunno.